CRM/Contact/BAO/Group.php

```diff
@@ -1123,11 +1123,12 @@ private static function buildGroupHierarchy(&$hierarchy, $group, $tree, $titleOn
       $hierarchy[$group['id']] = $spaces . $group['title'];
     }
     else {
-      $hierarchy[$group['id']] = [
-        'title' => $spaces . $group['title'],
+      $hierarchy[] = array(
+        'id' => $group['id'],
```

Comment:
The array() syntax seems to have come back. While I personally prefer array() the civi standard is now square brackets.